### ISSUE TITLE
Arm Support

### ISSUE THREAD

**Issue by jarodium** [nasa/QuIP#1]
Hello 

Does it support arm arch?

best regards

**Comment by jbmulligan** [comment]
On 6/30/16 9:03 AM, jarodium wrote:
>
> Does it support arm arch?
>
>
I'm not quite sure I understand the question.  Apple's iOS devices use 
ARM processors to the best of my knowledge, so in that sense the answer 
is yes.  I would rephrase the questions as, does your C compiler support 
your target architecture?

Some time ago I added support for intel SSE instruction set (using bits 
of assembler).  Now gcc supports SSE acceleration; I don't know whether 
or not gcc or other compilers support the ARM SIMD instruction set.

-Jeff

**Comment by jarodium** [comment]
Hello @jbmulligan 
Thank you for the reply.
I meant if the software can run on the latest Raspberry Pi.

Regards
Pedro

**Comment by jbmulligan** [comment]
On 6/30/16 12:22 PM, jarodium wrote:
> I meant if the software can run on the latest Raspberry Pi.

I haven't tried, but my understanding is that these things run a linux 
kernel, and presumably have a C compiler available, so my guess is yes.

-Jeff

**Comment by jarodium** [comment]
Ok
I will take a test run on it, when I need it : )

Thank you for your time
Regards
Pedro
